Cargo.toml

```diff
@@ -18,7 +18,7 @@ serializable = ["serde"]
 [dependencies]
 markdown = "1.0.0-alpha.4"
 serde = { version = "1", optional = true }
-swc_core = { version = "0.44.0", features = [
+swc_core = { version = "0.43.23", features = [
```

Comment:
I'm a bit confused.
To get fixes, we need to go from `0.44` back to `0.43`?

Reply:
oh wait, something is not right 🤔

Reply:
Yes I think you're right, there's something weird happening on my end while working on bump. Thanks for catching.